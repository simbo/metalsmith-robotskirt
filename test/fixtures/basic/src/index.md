
# A Markdown Post

I am **so** markdown.

Hey, *this* is `code` with ÚŦF châracters!

And _here_ some __other__ "stuff".

## And now...

- something
- completely
- different...
