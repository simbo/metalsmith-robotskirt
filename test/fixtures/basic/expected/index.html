<h1>A Markdown Post</h1>

<p>I am <strong>so</strong> markdown.</p>

<p>Hey, <em>this</em> is <code>code</code> with ÚŦF châracters!</p>

<p>And <em>here</em> some <strong>other</strong> &ldquo;stuff&rdquo;.</p>

<h2>And now&hellip;</h2>

<ul>
<li>something</li>
<li>completely</li>
<li>different&hellip;</li>
</ul>
